pkg/controller/resource/daemonset.go

```diff
@@ -28,8 +28,8 @@ import (
 )
 
 // CheckDaemonSetAnnotation will check the annotation of DaemonSet
-func CheckDaemonSetAnnotation(clientSet *kubernetes.Clientset, engine *chaosTypes.EngineInfo) (*chaosTypes.EngineInfo, error) {
-	targetAppList, err := getDaemonSetLists(clientSet, engine)
+func CheckDaemonSetAnnotation(clientset kubernetes.Interface, engine *chaosTypes.EngineInfo) (*chaosTypes.EngineInfo, error) {
```

Comment:
@ispeakc0de If you don't mind can we push these changes in separate PR? Creating a lot of confusion.

Reply:
These changes are minor but dependent changes, without these changes unit tests, fail so modified in the same PR. Changes are straight forward passing the client from the controller instead of declaring it inside the resource file.